modules/cnf-deploying-the-du-infrastructure-profile.adoc

```diff
@@ -0,0 +1,267 @@
+// CNF-950 4.7 Deploying the DU infrastructure profile
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
```

Comment:
This should be the full path of the assembly scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc

Reply:
Approved.

Reply:
@StephenJamesSmith I don't see this change updated in the file. Do you need to re-push your edits?